A37-xds-aggregate-and-logical-dns-clusters.md

```diff
@@ -112,6 +119,11 @@ problem, we might be able to fix this later via subchannel pooling
 functionality).  However, we will share the atomic used for circuit breaking
 thresholds across all connections for a given cluster, as discussed below.
 
+If any of a CDS policy's watchers reports that the resource does not exist, the
```

Comment:
Please also add a paragraph indicating that the tree of aggregate clusters must never be deeper than 16.  If the tree is deeper than that, then the CDS policy will report that it is in state TRANSIENT_FAILURE.

Reply:
Added